__tests__/index.test.js

```diff
@@ -0,0 +1,11 @@
+import '@testing-library/jest-dom';
```

Comment:
I'd double-check with @dustin-jw  on the location of these and whether or not we'd want to have tests in their dedicated directory or sibling to the file they test.

Reply:
I think we have been putting tests with their sibling files. Since it's not specified on the Jira card, it's probably going to be this way here too. Good idea to check w/ @dustin-jw though

Reply:
I did ask him and I'll rearrange for sibling files.